generator/src/test/java/com/scottlogic/deg/generator/cucumber/AValid.feature

```diff
@@ -246,106 +256,169 @@ Scenario: Running an 'aValid' request that includes a value of a string "ISIN" s
     Then the following data should be generated:
       | foo            |
       | "GB0002634946" |
+      | null           |
     Examples:
       | length |
       | 0      |
       | 1      |
       | 11     |
 
-  Scenario: Running an 'aValid' request alongside a contradicting longerThan constraint should fail with an error message
+  @ignore @bug # issue raised 246
+  Scenario: Running an 'aValid' request alongside a contradicting longerThan constraint should generate null
     Given foo is a valid "ISIN"
-    And foo is longer than 12
-    Then I am presented with an error message
-    And no data is created
+      And foo is longer than 12
+    Then the following data should be generated:
+      | foo  |
+      | null |
 
-  @ignore @bug
+  @ignore @bug # issue raised 246
   Scenario: Running an 'aValid' request alongside a non-contradicting shorterThan constraint should be successful
     Given foo is a valid "ISIN"
-    And foo is shorter than 13
-    And foo is in set:
-      | "GB0002634946" |
+      And foo is shorter than 13
+      And foo is in set:
+        | "GB0002634946" |
     Then the following data should be generated:
+      | foo            |
       | "GB0002634946" |
+      | null           |
 
-  Scenario: Running an 'aValid' request alongside a contradicting shorterThan constraint should fail with an error message
+  @ignore @bug # issue raised 246
+  Scenario: Running an 'aValid' request alongside a contradicting shorterThan constraint should generate null
     Given foo is a valid "ISIN"
-    And foo is shorter than 12
-    Then I am presented with an error message
-    And no data is created
+      And foo is shorter than 12
+    Then the following data should be generated:
+      | foo  |
+      | null |
 
-  @ignore @bug
+    ########### is this a duplicate or is the test content incorrect? ############
```

Comment:
The test is to validate that aValid can be run against a second aValid constraint in the same rule (L13 in the test matrix). this appears to be a duplicate because the second aValid has been removed.

Reply:
Ah, apologies, will add this back in.